tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
In my environment(VSCode), there're some problems about tsconfig.json after I run `yarn dev` in tab called Problems:

`Cannot write file because it would overwrite input file`

Only when I delete `"allowJs": true,`, the warnings disappear.

Reply:
You might need to add an outDir in tsconfig. Any outDir works.

Reply:
tsc or vue-tsc is only for type checking. So disable the output by setting noEmit should be able to solve this issue. Please try to restart the IDE to see if the error resolved.